File: test/test_3.js
console.log('Please input a string:')
str=prompt()
l=str.length
flag=0
for(i = 0; i < l; i++){
    if(str[i]!==str[l - 1 - i]){
        flag=1
        break
    }
}
if(flag){
    console.log('False')
}
else{
    console.log('True')
}

File: test_3.py
while(1):
    print("Please input a string:\n")
    str = input()
    l = len(str)
    flag  = 0
    for i in range(0,l):
        if(str[i] != str[l-1-i]):
            flag = 1
            break
    if(flag):
        print("This is not a palindrome string.\n")
    else:
        print("This is a palindrome string.\n")
